convert2rhel/main.py

```diff
@@ -75,11 +75,12 @@ def main():
         process_phase = ConversionPhase.PRE_PONR_CHANGES
         pre_ponr_conversion()
 
-        loggerinst.warning("The tool allows rollback of any action until this"
-                           " point.")
-        loggerinst.warning("By continuing all further changes on the system"
-                           " will need to be reverted manually by the user,"
-                           " if necessary.")
+        loggerinst.warning("The tool allows rollback of any action until this" " point.")
```

Comment:
Only bug I've seen so far is the lack of handling of multi-line strings, here it doesn't merge the strings but just brings them to the same line